panel/auth.py

```diff
@@ -450,10 +451,17 @@ def _raise_error(self, response, body=None, status=400):
             log.warning(f"{provider} OAuth provider failed to fully "
                         f"authenticate returning the following response:"
                         f"{body}.")
+        if hasattr(body, "get"):
```

Comment:
In my case `body` was `None` due to the decoding error triggering an `AttributeError`.